### ISSUE TITLE
Source gen fails when a boolean param is used in a class

### ISSUE THREAD

**Issue by oldominion** [b-straub/DexieNET#3]
Not sure if this is only under certain circumstances, (e.g. classes instead of records), but this effectively reproduces the issue. I can provide a reproduction project if you'd like.

```C#
builder.Services.AddDexieNET<MultiTableDB>();
```

```C#
   [DBName("MultiTableDB")]
   public interface IMultiTableStore : IDBStore { };

   [Schema(StoreName = "Users")]
   public partial class UsersEntity : IMultiTableStore
   {
       [Index(IsPrimary = true)]
       public string ID { get; set; } = "";

       //Source gen fails when this is uncommented
       public bool param1 { get; set; }

       protected UsersEntity() { }
   }
   ```

**Comment by oldominion** [comment]
Same with records.

```C#
    [Schema(StoreName = "Persons")]
    public partial record PersonEntity
    (
        [property: Index] string Name,
        [property: Index] int Age,
        bool Alive // fail
    ) : IMultiTableStore;
```

**Comment by b-straub** [comment]
Thank you for your report. I have fixed this in the cloud branch, but I need to check if porting back is possible. As workaround you can just index the boolean as shown here: 
https://github.com/b-straub/DexieNET/blob/ce476f6748dc3cca925b5c438b0f5d153a0a4f7a/DexieNETTest/TestBase/Test/Data/DBStores.cs#L98

this will invoke the needed JSON converter.

**Comment by b-straub** [comment]
Fixed with 1.0.2, please check and close when working.
